myuw/templates/handlebars/card/bothell_application.html

```diff
@@ -0,0 +1,66 @@
+{% load templatetag_handlebars %}
+{% tplhandlebars "bothell_application_card" %}
+<div class="myuw-card application-card">
+{{#if is_returning}}
+    <h3 class="myuw-card-title">
+        Your Returning Student application
+    </h3>
+
+    <div class="myuw-card-section last">
+		<h4 class="applicant-card myuw-header-msg"> For application status, contact the UW Bothell Office of Admissions</h4>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Email</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content"><a href="mailto:uwbifo@uw.edu">uwbifo@uw.edu</a></div>
+                </div>
+            </div>
+			<div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Phone</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">(425) 352-5000</div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                    <div class="myuw-card-row-heading">Mail</div>
+                </div>
+                <div class="col-sm-9">
+                    <div class="myuw-card-row-content">10909 NE 185th St <br>Bothell, WA 98011<br>Box 358500 </div>
+                </div>
+            </div>
+	</div>
+	<div class="myuw-card-section">
+		<div class="links-container actionlink-list">
+			<h4> Resources for Bothell Applicants</h4>
+				<ul class="unstyled-list list-leading ">
+				    <li><a href="http://www.uwb.edu/financialaid" target="_blank">Learn all about student financial aid and scholarships</a></li>
```

Comment:
Make sure you include the `data-linklabel` attribute on all the links you want to control the link text for when it is added to the "Recently Visited" section.

Reply:
Got it. Didn't know that's for Quicklinks.